addons/im_livechat/static/src/legacy/public_livechat.js

```diff
@@ -177,21 +177,22 @@ var LivechatButton = Widget.extend({
      * @param {Array} notification
      */
     _handleNotification: function (notification) {
-        if (this._livechat && (notification[0] === this._livechat.getUUID())) {
-            if (notification[1]._type === 'history_command') { // history request
-                var cookie = utils.get_cookie(LIVECHAT_COOKIE_HISTORY);
-                var history = cookie ? JSON.parse(cookie) : [];
+        const [livechatUUID, notificationData] = notification;
+        if (this._livechat && (livechatUUID === this._livechat.getUUID())) {
+            if (notificationData._type === 'history_command') { // history request
+                const cookie = utils.get_cookie(LIVECHAT_COOKIE_HISTORY);
+                const history = cookie ? JSON.parse(cookie) : [];
                 session.rpc('/im_livechat/history', {
                     pid: this._livechat.getOperatorPID()[0],
                     channel_uuid: this._livechat.getUUID(),
                     page_history: history,
                 });
-            } else { // normal message
+            } else if (notificationData.body) { // normal message
```

Comment:
`body in notificationData`? Can we potentially get falsy value? Eg. only attachment, is body empty or <p> stuff?

Reply:
I think "empty" value is "<p></p>" but ok for changes